src/Model.js

```diff
@@ -823,7 +708,9 @@ class Model {
         if (DataType.name === 'Array' || DataType.name === 'Object') {
           attr = toJS(value)
         } else if (DataType.name === 'Date') {
-          attr = moment(value).toISOString()
+          attr = value instanceof Date
+            ? value
+            : new Date(Date.parse(value)).toISOString()
```

Comment:
If `value` is a moment object here will it automatically coerce it to a string that is parseable by `Date.parse`?